Microsoft.Health.Fhir.sln

```diff
@@ -137,6 +137,10 @@ Project("{D954291E-2A0B-460D-934E-DC6B0785DB48}") = "Microsoft.Health.Fhir.Share
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "LogsAndInterpolations", "tools\FhirAnalyzer\FhirAnalyzer.csproj", "{389C844A-4DC2-4780-9821-464580396850}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Health.Fhir.TaskManagement", "src\Microsoft.Health.Fhir.TaskManagement\Microsoft.Health.Fhir.TaskManagement.csproj", "{03AD4FC4-A56F-4E94-B295-B02A9E3E3CCD}"
```

Comment:
If this logic doesn't have anything to do with FHIR then it could just be `Microsoft.Health.TaskManagement` and sounds like it could be something that is later moved to Shared Components

Reply:
Updated to taskmanagement.